src/saving/SaveHelper.cs

```diff
@@ -131,11 +129,48 @@ private static Save CreateSaveObject(MainGameState gameState, SaveInformation.Sa
         };
     }
 
+    private static string BuildQuicksaveFilename(uint quicksaveId)
+        => $"quick_save.{quicksaveId}.{Constants.SAVE_EXTENSION}";
+
+    /// <summary>
+    ///   Goes through the save-folder to detect the last quicksave-id
+    /// </summary>
+    private static uint GetLastQuicksaveId()
+    {
+        using (var directory = new Directory())
+        {
+            directory.Open(Constants.SAVE_FOLDER);
+            for (uint i = 0; ; i++)
+            {
+                if (!directory.FileExists(BuildQuicksaveFilename(i)))
+                    return i - 1;
```

Comment:
I don't think this is good. Instead the existing files should be enumerated.

Reply:
Also the file update timestamps should be used to detect which save was last modified. Quick load should also detect autosaves and manual saves as possibly being the last one. So quick load should load the save file that has the latest timestamp despite whatever the file name might be.